### ISSUE TITLE
[Feature] Use labels for basic-health pod

### ISSUE THREAD

**Issue by mpostument** [kubecost/cost-analyzer-helm-chart#3115]
### Problem Statement

I am not able to deploy cost-analyzer to kubernetes because kyverno policies expect for few labels to be present in basic-health pod

```shell
check-for-labels: 'validation error: The label `[app.kubernetes.io/name`] is required.
    rule check-for-labels failed at path /metadata/labels/'
```

### Solution Description

1. Use same labels for basic health pods as for other cost-analyzer components

### Troubleshooting

- [X] I have read and followed the [issue guidelines](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/ISSUE_GUIDELINES.md) and this is a feature request only for the Helm chart.
- [X] I have searched other issues in this repository and mine is not recorded.

**Comment by chipzoller** [comment]
That appears to be a Kyverno policy blocking the deployment. What version of Kubecost are you deploying and with what values?

**Comment by chipzoller** [comment]
Sent fix in #3118
